.github/workflows/update-beats.yml

```diff
@@ -34,11 +34,13 @@ jobs:
       - uses: elastic/apm-pipeline-library/.github/actions/updatecli@current
         env:
           BRANCH_NAME: ${{ matrix.branch }}
+          GITHUB_ACTOR: ${{ github.actor }}
         with:
           vaultUrl: ${{ secrets.VAULT_ADDR }}
           vaultRoleId: ${{ secrets.VAULT_ROLE_ID }}
           vaultSecretId: ${{ secrets.VAULT_SECRET_ID }}
           pipeline: ./.ci/update-beats.yml
+          command: '--experimental apply'
```

Comment:
I'm wondering if it makes sense to set the `--experimental` flag in the action itself? Would it be backwards compatible?

Reply:
I was thinking to support a flag, but since command is a way to pass the different arguments to the updatecli binary, I think it's fine.

`--experimental` is a flag supported by the updatecli for quite some time, so I think it's backwards compatible. Or  what do you mean?

Reply:
My thought was, if we set it by default in in the action, we don't have to set in every action usage.

Reply:
Also, it might be easier to migrate in the future if it becomes "non-experimental"

Reply:
No, `--experimental` feature by default it might produce some side-effects, see this [comment](https://github.com/updatecli/updatecli/issues/1914#issuecomment-2081151292); therefore I prefer to be explicit when to use it for the shake of reliability

Reply:
I see. Thanks for the explanation.